tests/qos/test_qos_sai.py

```diff
@@ -260,11 +273,27 @@ def testPfcStormWithSharedHeadroomOccupancy(
         except Exception as e:
             raise e
         finally:
+            if prev_poll_interval:
+                logger.info("--- Restore original poll interval {} ---".format(prev_poll_interval))
+                duthost.command("pfcwd interval {}".format(prev_poll_interval))
+            else:
+                logger.info("--- Set Default Polling Interval ---".format())
+                duthost.command("pfcwd interval {}".format("200"))
+
+            if prev_state:
+                logger.info("--- Restore original config {} for PfcWd on {} ---".format(prev_state, pfcwd_test_port))
+                start_wd_on_ports(duthost,
+                        pfcwd_test_port,
+                        prev_state.get("restoration_time", "200")
+                        prev_state.get("detection_time", "200"),
+                        prev_state.get("action", "drop"))
```

Comment:
if pfcwd was running prior to the test, the cleanup should be "pfcwd start_default" which will ensure that pfcwd is started with the default restoration/detection time for each platform. The restoration/detection times are not always 200ms. They vary across platforms